packages/core/src/App/AppContent.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { DesktopWrapper } from '@deriv/components';
+import { useFeatureFlags } from '@deriv/hooks';
+import { observer } from '@deriv/stores';
+import Wallets from '@deriv/wallets';
+import BinaryBotIFrame from 'Modules/BinaryBotIFrame';
+import SmartTraderIFrame from 'Modules/SmartTraderIFrame';
+import { Route } from 'react-router-dom';
+import ErrorBoundary from './Components/Elements/Errors/error-boundary.jsx';
+import AppContents from './Containers/Layout/app-contents.jsx';
+import Footer from './Containers/Layout/footer.jsx';
+import Header from './Containers/Layout/header';
+import AppModals from './Containers/Modals';
+import PlatformContainer from './Containers/PlatformContainer/PlatformContainer.jsx';
+import Routes from './Containers/Routes/routes.jsx';
+import AppToastMessages from './Containers/app-toast-messages.jsx';
+import Devtools from './Devtools';
+
+const AppContent: React.FC<{ passthrough: unknown }> = observer(({ passthrough }) => {
+    const { is_next_wallet_enabled } = useFeatureFlags();
+
+    if (is_next_wallet_enabled)
+        return (
+            <div style={{ width: '100vw', height: '100vh' }}>
+                <Route path={'/'} component={Wallets} />
+                <Devtools />
+            </div>
+        );
```

Comment:
When the `is_next_wallet_enabled` is enabled, Replace the whole legacy app with a single route that renders `@deriv/wallets`.